code/model/editableformfields/EditableMultipleOptionField.php

```diff
@@ -171,7 +177,7 @@ public function getHasAddableOptions() {
 	 */
 	protected function getOptionsMap() {
 		$optionSet = $this->Options();
-		$optionMap = $optionSet->map('EscapedTitle', 'Title');
+		$optionMap = $optionSet->map('Value', 'Title');
```

Comment:
yeah, I was thinking about that. The only issue is that an empty option can also be valid so may cause issues going forward too.

I'll let you make a call and i'll update accordingly.